### DISCUSSION TITLE
Διάδραση ανθρώπου / υπολογιστή, με… πλέξιμο;

### DISCUSSION THREAD

**Discussion by mariosconsta** [courses-ionio/hci#1728] [Announcements]
Κάτι που βρήκα υπερβολικά ενδιαφέρον για τη διάδραση που είχε ο άνθρωπος με τους πρώτους υπολογιστές. Διαβάστε το όλο!
[TIL The Bra Ladies Got Us To The Moon](https://imgur.com/gallery/8DwbcBG)

**Comment by Sola1r3** [comment]
Πολύ ενδιαφέρον. Κατι άλλο εξίσου ενδιαφέρον με ένα πολύ ωραίο όραμα για την διάδραση και σχετικό με το μάθημα και τις ιδέες του Engelbart είναι το [dynamicland](https://dynamicland.org/). Αξίζει να το ρίξετε μια ματιά για όσους δεν το ξέρουν ήδη, είναι πολυ δημιουργικό.

**Reply by epidrome** [reply]
πολύ καλό αλλά μήπως είναι νέο-άλλο θέμα?

**Comment by epidrome** [comment]
@mariosconsta 
για την άριστη επίδοση, καλό είναι να γίνεται και μια αναφορά στο υλικό του βιβλίο, αν υπάρχει κάτι σχετικό, π.χ.,:
https://pibook.epidro.me/gallery/card-puncher/
